packages/jest-util/src/install_common_globals.js

```diff
@@ -13,10 +13,26 @@ import type {Global} from 'types/Global';
 import createProcesObject from './create_process_object';
 import deepCyclicCopy from './deep_cyclic_copy';
 
+const DTRACE = [
```

Comment:
is there any module where this list is maintained?

Reply:
I have the same question.

Reply:
Yes, we can do something like `Object.keys(global).filter(dtrace => dtrace.startsWith('DTRACE_'))` to get it.